pgmpy/models/FunctionalBayesianNetwork.py

```diff
@@ -0,0 +1,177 @@
+import networkx as nx
+import numpy as np
+import pandas as pd
+
+from pgmpy.factors.hybrid import FunctionalCPD
+from pgmpy.global_vars import logger
+from pgmpy.models import BayesianNetwork
+
+
+class FunctionalBayesianNetwork(BayesianNetwork):
+    def add_cpds(self, *cpds):
+        """
+        Add Functional CPD (Conditional Probability Distribution)
+        to the Bayesian Network.
+
+        Parameters
+        ----------
+        cpds  :  instances of FunctionalCPD
+            List of FunctionalCPDs which will be associated with the model
+
+        Examples
+        --------
+        >>> from pgmpy.factors.hybrid.FunctionalCPD import FunctionalCPD
+        >>> from pgmpy.models.FunctionalBayesianNetwork import FunctionalBayesianNetwork
```

Comment:
Same here should be able to import `from pgmpy.factors.hybrid` and `pgmpy.models`